common/shlibs

```diff
@@ -30,6 +30,8 @@ ld64.so.2 glibc-2.32_1 ppc64
 ld.so.1 glibc-2.32_1 mips
 ld.so.1 glibc-2.32_1 ppc
 ld-linux-armhf.so.3 glibc-2.32_1
+libfwupd.so.2 fwupd-1.6.3_1
+libfwupdplugin.so.2 fwupd-1.6.3_1
```

Comment:
Are these just to fix the warnings? If nothing links against them, we don't need to add them.

Reply:
Yes. I will then remove them.